birdhouse/deprecated-components/canarie-api-full-monitoring-deprecated/.gitignore

```diff
@@ -0,0 +1,5 @@
+config/canarie-api/canarie_api_full_monitoring.py
+
+# Old paths. Keep these so that old config files remain uncommittable after updates.
```

Comment:
Contrary to the other `birdhouse/config/.gitignore` that ignores the old paths due to the `deprecated-components` refactor, this seems to be related to the separation of `canarie-api` and `proxy` into interdependent services. 

Is it the case, and if so, does it still apply here?
The new version of CanarieAPI to support this is not yet released.

Or is it simply that we let CanarieAPI define its own dependency to `proxy`, and we only depend on `canarieapi` here?

Reply:
These lines are still here because I just copied these files from the optional-components directory. They no longer need to be here so I will remove them. 

> The new version of CanarieAPI to support this is not yet released.

This change isn't affected by the new CanarieAPI version

> Or is it simply that we let CanarieAPI define its own dependency to proxy, and we only depend on canarieapi here?

I think that makes the most sense. You'll see that this component depends on the config/canarie-api component which can then depend on proxy if needed.